RestSharp/Authenticators/OAuth/OAuthTools.cs

```diff
@@ -105,9 +105,9 @@ public static string GetTimestamp(DateTime dateTime)
         /// The set of characters that are unreserved in RFC 2396 but are NOT unreserved in RFC 3986.
         /// </summary>
         /// <seealso cref="http://stackoverflow.com/questions/846487/how-to-get-uri-escapedatastring-to-comply-with-rfc-3986" />
```

Comment:
As another comment, I agree with these formatting changes, but it kinda feels weird to see them in this PR.